core/plugins/org.polarsys.capella.core.data.core.properties/src/org/polarsys/capella/core/data/core/properties/sections/TitleBlockCellSection.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * Copyright (c) 2020 THALES GLOBAL SERVICES.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *  
+ * Contributors:
+ *    Thales - initial API and implementation
+ *******************************************************************************/
+package org.polarsys.capella.core.data.core.properties.sections;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.jface.viewers.ISelection;
+import org.eclipse.sirius.viewpoint.description.DAnnotation;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.ui.IWorkbenchPart;
+import org.eclipse.ui.views.properties.tabbed.TabbedPropertySheetPage;
+import org.polarsys.capella.core.data.core.properties.fields.TitleBlockBasicElementGroup;
+import org.polarsys.capella.core.ui.properties.fields.AbstractSemanticField;
+import org.polarsys.capella.core.ui.properties.sections.AbstractSection;
+
+/**
+ * The PropertyValueGroup section.
+ */
+public class TitleBlockCellSection extends AbstractSection {
+  protected TitleBlockBasicElementGroup titleBlockBasicElementGroup;
+
+  /**
+   * @see org.eclipse.jface.viewers.IFilter#select(java.lang.Object)
+   */
+  @Override
+  public boolean select(Object toTest) {
+    EObject eObjectToTest = super.selection(toTest);
+
+    return ((eObjectToTest != null) && (eObjectToTest instanceof DAnnotation)
+        && ((DAnnotation) eObjectToTest).getSource().startsWith("TB"));
```

Comment:
Could you extract Keywords like "TB" into constants with meaningful name so that it's easier to maintain the code?

Reply:
Done.